profitbricks/src/main/java/org/jclouds/profitbricks/domain/DataCenter.java

```diff
@@ -49,81 +46,55 @@
 
    @Nullable
    public abstract List<Storage> storages();
-//   @Nullable public abstract List<LoadBalancer> loadBalancers();
-
-   public static DataCenter create(String id, String name, int version, ProvisioningState state, Location location, List<Server> servers,
-           List<Storage> storages) {
-      return new AutoValue_DataCenter(id, name, version, state, location,
-              servers != null ? ImmutableList.copyOf(servers) : Lists.<Server>newArrayList(),
-              storages != null ? ImmutableList.copyOf(storages) : Lists.<Storage>newArrayList());
-   }
 
+//   @Nullable public abstract List<LoadBalancer> loadBalancers();
    public static Builder builder() {
-      return new Builder();
+      return new AutoValue_DataCenter.Builder()
+              .servers(ImmutableList.<Server>of())
+              .storages(ImmutableList.<Storage>of());
    }
 
    public Builder toBuilder() {
-      return builder().fromDataCenter(this);
+      return new AutoValue_DataCenter.Builder(this);
    }
 
-   public static final class Builder {
-
-      private String id;
-      private String name;
-      private ProvisioningState state;
-      private Location location;
-      private int version;
-      private List<Server> servers;
-      private List<Storage> storages;
-//      private List<LoadBalancer> loadBalancer;
+   @AutoValue.Builder
+   public abstract static class Builder {
 
-      public Builder id(String id) {
-         this.id = id;
-         return this;
-      }
+      public abstract Builder id(String id);
 
-      public Builder name(String name) {
-         this.name = name;
-         return this;
-      }
+      public abstract Builder name(String name);
 
-      public Builder state(ProvisioningState state) {
-         this.state = state;
-         return this;
-      }
+      public abstract Builder version(int version);
 
-      public Builder location(Location location) {
-         this.location = location;
-         return this;
-      }
+      public abstract Builder state(ProvisioningState state);
 
-      public Builder version(int version) {
-         this.version = version;
-         return this;
-      }
+      public abstract Builder location(Location location);
 
-      public Builder servers(List<Server> servers) {
-         this.servers = servers;
-         return this;
-      }
+      public abstract Builder servers(List<Server> servers);
```

Comment:
To be more precise, should probably use autoBuild and copyOf, like other places in this PR?